content_api_doc.md

```diff
@@ -0,0 +1,42 @@
+Content API
```

Comment:
How about putting this (and all the other documentation that's not README) into a `docs/` directory?  As @philschatz said, we can link to it in README.